app/views/users/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<section class="user-infor">
+  <div class="user">
+    <div>
+      <img src="<%= @user.photo %>" alt="Profile picture" />
+    </div>
+    <div class="user-details">
+      <h2><%= @user.name %></h2>
+      <span>Number of posts: <%= @user.posts_counter %></span>
+    </div>
+  </div>
+  <div class="user-bio">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <ul class="user-posts">
+    <% @user.recent_posts.each do |post| %>
+      <li>
+        <h4><%= post.title %></h4>
+        <a href="<%= @user.id%>/posts/<%= post.id%>">
+          <p><%= post.text.split[0..30].join(' ') %>....</p>
+          <div class="user-count">
+            <span>Comments: <%= post.comments_counter %></span>
+            <span>Likes: <%= post.likes_counter %></span>
+          </div>
```

Comment:
- [ Required ] I think it would be better if you align this part of the users show page (comments / likes) just the same way it is illustrated on the project wireframe. It should be aligned to the `right` no the left. Please see the screenshot below and rectify this.

## Your design
![Screenshot from 2023-07-08 04-20-53](https://github.com/Rana-Mudassir/blog-app/assets/16392046/812308ed-87b3-4334-ac32-023605f11da0)

## Project wireframe
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABTYBUDYHTRJFZ7G6LBYWR26ZFIZGRQ)

Reply:
I updated the Comments/Likes part according to the wireframe. Please check the attached Image.
![updated](https://github.com/Rana-Mudassir/blog-app/assets/63316090/cbb07ec3-bdb2-4af3-b1ef-1c47e2de116f)